example/src/App.js

```diff
@@ -64,6 +64,19 @@ function App() {
   const [backendUrl] = useState(process.env.REACT_APP_REST_BASE_URL);
   const [hasDefaultUrl, setHasDefaultUrl] = useState(false);
 
+  useEffect(() => {
+    const currentUrl = window.location.href;
+    // Open the drawer when in REST mode and no uuid is present.
+    // Note: The lowest subdirectory of the url is extracted to see if
+    // it contains a uuid.
+    if (
+      isRestMode &&
+      currentUrl.substring(currentUrl.lastIndexOf("/") + 1) === ""
+    ) {
+      setIsDrawerOpen(true);
+    }
+  }, [isRestMode]);
```

Comment:
I want to note this in this PR: this is probably a hacky solution to extract the uuid in `App.js` to open the drawer, but one of the better approaches for now. Another method would be to use `useParams()` in `App.js` to grab the uuid. Although this doesn't seem to work, due to the timing complexity of `OSCALLoader` working with `App.js`. If we end up making changes to how the url displays the uuid, we will probably need to update this.